mitosheet/mitosheet/public/README.md

```diff
@@ -0,0 +1,8 @@
+# Public Interfaces
+
+Public interfaces are what are exposed to users. With Mito, we do our _very best_ to never break any code generates by Mito, which requires maintaining all old public interfaces we've ever supported. While there's a minor maintaince burden, we aim to make upgrading as easy as possible.
```

Comment:
Yeah, one thing we could call out here is that "break any code" doesn't mean just that the code errors, but it's that the result of running the code on the same data changes.